### PR TITLE
feat : added order summary

### PR DESCRIPTION
**Pull request by rutvikpumak**
-> after payment it will show order summary page
-> click on checkout , made dummy payment and test it

### CODE REVIEW

File: src/context/auth/authContext.js [hunk 1]
```diff
@@ -57,25 +57,14 @@ const AuthProvider = ({ children }) => {
     }
   };
 
-  // useEffect(() => {
```
Comment (Rohanmond) [inline src/context/auth/authContext.js:60]: remove unwanted comments

File: src/pages/Checkout/CheckoutPrice.js [hunk 2]
```diff
@@ -69,7 +76,7 @@ export function CheckoutPrice({ setMsg }) {
     if (address.length === 0) {
       toast.error("Please Add Address");
       setTimeout(() => {
-        navigate("/userProfile");
+        navigate("/user_profile");
```
Comment (Ash-D23) [inline src/pages/Checkout/CheckoutPrice.js:79]: Avoid using settimeout this way... instead you can use a async function and redirect

**Review by Ash-D23 — APPROVED** [review]
- Clean Code
-  Modularized well
-  Good use of Context

**Review by Rohanmond — APPROVED** [review]
Folder structure looks good 
code is looking good to merge

**Pull request merged**